### ISSUE TITLE
All ListView items are being built at once

### ISSUE THREAD

**Issue by RasitAyaz** [bnxm/material_floating_search_bar#41]
I want to display a `ListView` as search results inside the body of my search bar. I set `physics` to the `NeverScrollableScrollPhysics` and `shrinkWrap` to `true` as described in the usage documentation.

```
builder: (context, transition) {
  return ClipRRect(
    borderRadius: BorderRadius.circular(largeRadius),
    child: Material(
      color: primaryColor,
      elevation: 4.0,
      child: ListView.builder(
        physics: NeverScrollableScrollPhysics(),
        shrinkWrap: true,
        itemBuilder: (context, index) => buildSearchItem(index),
        itemCount: _results.length,
      ),
    ),
  );
},
```

My list items load images from the web so I want just visible items to be built, not all items in the list. This is normally default behaviour of the `ListView.builder`, but search bar builds all items in the list at once (visible or not).

**Comment by bnxm** [comment]
Yes, this happens when you set `shrinkWrap` to true. If you want to use infinite lists, you'll have to set the `isScrollControlled` flag on the search bar to true.

**Comment by RasitAyaz** [comment]
That didn't help unfortunately, still loading at once. :(

**Comment by armandojimenez** [comment]
Anyone to deal with this?
